op-nat/nat.go

```diff
@@ -85,47 +118,157 @@ func (n *nat) Stopped() bool {
 func (n *nat) printResultsTable(runID string) {
 	n.config.Log.Info("Printing results...")
 	t := table.NewWriter()
-	t.SetStyle(table.StyleColoredBlackOnGreenWhite)
 	t.SetOutputMirror(os.Stdout)
-	t.SetTitle("NAT Results")
-	t.AppendHeader(table.Row{"Type", "ID", "Result", "Error"})
-	colConfigAutoMerge := table.ColumnConfig{AutoMerge: true}
-	t.SetColumnConfigs([]table.ColumnConfig{colConfigAutoMerge})
-
-	var overallResult ResultType = ResultPassed
-	var hasSkipped = false
-	var overallErr error
-	for _, res := range n.results {
-		appendResultRows(t, res)
-		overallErr = errors.Join(overallErr)
-		if res.Result == ResultFailed {
-			overallResult = ResultFailed
+	t.SetTitle(fmt.Sprintf("NAT Results (%s)", formatDuration(n.result.Duration)))
```

Comment:
It would be nice to separate the raw data from its formatting (I can guarantee we'll want a json output in 3... 2.. 1.. :))

Reply:
Isn't it already though? This function, `printResultsTable`, simply iterates over `nat.result` (the data). We could write a similar function `printJSON` by iterating over the same results data.
Or am I missing something?